Description: install binary in /usr/bin instead of /usr/local/bin
Author: Ludovic Rousseau <rousseau@debian.org>
--- a/makefile
+++ b/makefile
@@ -24,4 +24,4 @@
 	rm -f $(objs) jhead
 
 install:
-	cp jhead ${DESTDIR}/usr/local/bin/
+	cp jhead ${DESTDIR}/usr/bin/
